src/libespeak-ng/speech.c

```diff
@@ -118,17 +118,14 @@ static int dispatch_audio(short *outbuf, int length, espeak_EVENT *event)
 			voice_samplerate = event->id.number;
 
 			if (out_samplerate != voice_samplerate) {
-#ifdef HAVE_PCAUDIOLIB_AUDIO_H
+#if USE_LIBPCAUDIO
 				if (out_samplerate != 0) {
 					// sound was previously open with a different sample rate
 					audio_object_close(my_audio);
 					out_samplerate = 0;
-#ifdef HAVE_SLEEP
-					sleep(1);
-#endif
```

Comment:
I'd say make this a separate commit, we don't know what impact it may have

Reply:
Ah, configure actually wasn't looking for the sleep function, so this was indeed dead, ok.

Reply:
HAVE_SLEEP does not defined from any sources (including autoconf). It is indeed dead code.